packages/bundler/src/BundlerCollectorTracer.ts

```diff
@@ -214,12 +233,19 @@ export function bundlerCollectorTracer (): BundlerCollectorTracer {
         return
       }
 
+      const lastOpInfo = this.lastThreeOpcodes[this.lastThreeOpcodes.length - 2]
       // store all addresses touched by EXTCODE* opcodes
-      if (opcode.match(/^(EXT.*)$/) != null) {
-        const addr = toAddress(log.stack.peek(0).toString(16))
+      if (lastOpInfo?.opcode?.match(/^(EXT.*)$/) != null) {
+        const addr = toAddress(lastOpInfo.stackTop3[0].toString(16))
         const addrHex = toHex(addr)
+        const last3opcodesString = this.lastThreeOpcodes.reduce((a, b) => a + b.opcode + ' ', '')
```

Comment:
`last3opcodesString = lastThreeOpcodes.map(x=>x.opcode).join(" ")`

(and it has no trailing space)

Reply:
Fixed.